rst_to_myst/markdownit.py

```diff
@@ -619,10 +622,15 @@ def visit_DirectiveNode(self, node):
             and len(node.children) == 2
         ):
             # special case, where we can use standard Markdown fences
+            token_type = "fence"
+            directive_name = "code"
+            if self.code_to_code_cell:
+                token_type = "directive"
```

Comment:
@chrisjsewell I can't find the documentation on `token_types` -- so I don't see how to specify a `myst` directive when adding this token.